sefaria/helper/schema.py

```diff
@@ -306,13 +307,16 @@ def change_parent(node, new_parent, place=0):
     old_parent.children = [n for n in old_parent.children if n.key != node.key]
     new_parent.children.insert(place, node)
     node.parent = new_parent
-    new_normal_form = node.ref().normal()
+    new_normal_form = node.ref(force_update=True).normal()
 
     index.save(override_dependencies=True)
     library.rebuild()
 
     for link in linkset:
-        link.refs = [ref.replace(old_normal_form, new_normal_form) for ref in link.refs]
+        if exact_match:
```

Comment:
Seems there's a broader issue here of asking why link refs that simply contain the original title are all indiscriminately re-written?